eisbuk-admin/src/components/atoms/IntervalCard/IntervalCard.tsx

```diff
@@ -105,8 +76,13 @@ const BookingCard: React.FC<Props> = ({
           </Typography>
         </Box>
         <Box display="flex" flexGrow={1} flexDirection="column">
-          <Box display="flex" flexGrow={1} className={classes.topWrapper}>
-            {/* {timeSpan} */}
+          <Box
+            display="flex"
+            flexGrow={1}
+            className={classes.topWrapper}
+            onClick={handleClick}
```

Comment:
We’re clicking on the button, not the card, so remove this. If you check the code down below, you’ll see that handler is already assigned to the button